### ISSUE TITLE
IllegalStateException: The query result was empty, but expected a single row to return a NON-NULL object

### ISSUE THREAD

**Issue by autoreleasefool** [autoreleasefool/approach#550]
```
java.lang.IllegalStateException: The query result was empty, but expected a single row to return a NON-NULL object of type <ca.josephroque.bowlingcompanion.core.model.BowlerDetails>.
    at ca.josephroque.bowlingcompanion.core.database.dao.BowlerDao_Impl$getBowlerDetails$1.call(BowlerDao_Impl.kt:356)
    at ca.josephroque.bowlingcompanion.core.database.dao.BowlerDao_Impl$getBowlerDetails$1.call(BowlerDao_Impl.kt:338)
    at androidx.room.CoroutinesRoom$Companion$createFlow$1$1$1.invokeSuspend(CoroutinesRoom.kt:129)
    at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
    at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
    at java.lang.Thread.run(Thread.java:1012)
```

**Comment by autoreleasefool** [comment]
Might be resolved, this is a pretty old crash from `4.0.2`

**Comment by autoreleasefool** [comment]
Closing, likely not actionable
